### PR TITLE
[VPC] vpc module

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
You can create vpc and subnets for this vpcs

### CODE REVIEW

File: modules/vpc-peering/README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ variable "main_vpc" {
 }
 
 module "peering" {
-  source = "../../modules/vpc-peering"
+  source = "opentelekomcloud/modules/opentelekomcloud//modules/vpc-peering"
```
Comment (vladimirvshivkov) [inline modules/vpc-peering/README.md:61]: double slashes

File: examples/vpc/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+## Requirements
+
+| Name                                                                                           | Version    |
+|------------------------------------------------------------------------------------------------|------------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform)                      | >= 0.12.21 |
+| <a name="requirement_opentelekomcloud"></a> [opentelekomcloud](#requirement\_opentelekomcloud) | >= 1.23.9  |
+
+## Providers
+
+No providers.
```
Comment (vladimirvshivkov) [inline examples/vpc/README.md:26]: delete this maybe?

**Review by vladimirvshivkov — CHANGES_REQUESTED** [review]
you think that user should not have option to choose vpc names?

File: modules/dns/README.md [hunk 3]
```diff
@@ -71,7 +71,7 @@ locals {
 }
 
 module "dns" {
-  source = "../../modules/dns"
+  source = "opentelekomcloud/modules/opentelekomcloud//modules/dns"
```
[the remark this answers is not in the document]
Comment (anton-sidelnikov) [reply modules/dns/README.md:74]: exact that it works with remote hashi registry

File: examples/vpc/README.md (hunk 2 above)
Comment (anton-sidelnikov) [reply examples/vpc/README.md]: ok

**Review by artem-lifshits — APPROVED** [review]

**Pull request merged**